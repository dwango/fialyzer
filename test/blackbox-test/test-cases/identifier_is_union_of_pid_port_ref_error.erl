-module(identifier_is_union_of_pid_port_ref_error).
-export([main/0]).

-spec wrong_gen_ident() -> identifier().
wrong_gen_ident() ->
    false.

main() ->
    wrong_gen_ident().
