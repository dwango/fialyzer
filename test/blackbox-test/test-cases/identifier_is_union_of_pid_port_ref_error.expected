test-cases/identifier_is_union_of_pid_port_ref_error.erl:-1: wrong_gen_ident: Type spec unmatch;
  success type: fun(() -> 'false')
  type spec   : fun(() -> pid() | port() | reference())
